src/pmlogger/pmlogger_check.sh

```
@@ -534,7 +540,17 @@ _check_archive()
 	    :
 	else
 	    logdir=`dirname "$logfile"`
-	    echo "Directory (`cd "$logdir"; $PWDCMND`) contents:"
+	    # being cautious in case cd fails ...
+	    #
+	    __pwddir=`cd "$logdir && $PWDCMND`
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this simple command.

(SC1009)